src/main/java/sirius/web/templates/pdf/TagliatellePDFContentHandler.java

```diff
@@ -53,14 +55,37 @@ public boolean generate(Generator generator, OutputStream out) throws Exception
         ITextRenderer renderer = new ITextRenderer();
         renderer.getSharedContext()
                 .setReplacedElementFactory(new ImageReplacedElementFactory(renderer.getOutputDevice()));
-        renderer.setDocumentFromString(content);
+        renderer.setDocumentFromString(cleanHtml(content));
```

Comment:
**Optional:** Maybe this could be enabled optionally? Don't know if it could break stuff in S2

Reply:
Hmm, how? As an additional pseudo-parameter in the context? As a system-wide setting? Both do not seem to be particularly elegant 😇

Let's see what S2 is going to say about this PR, okay? We can also flag it as _breaking_, but it should process all previously valid input as it was, and only affect the behaviour in case of faulty input.